dev/benchmarks/platform_channels_benchmarks/android/app/src/main/kotlin/com/example/platform_channels_benchmarks/MainActivity.kt

```diff
@@ -23,7 +23,7 @@ class MainActivity: FlutterActivity() {
         } }
         val basicStandard = BasicMessageChannel(flutterEngine.dartExecutor, "dev.flutter.echo.basic.standard", StandardMessageCodec.INSTANCE)
         basicStandard.setMessageHandler { message, reply -> reply.reply(message) }
-        val basicBinary = BasicMessageChannel(flutterEngine.dartExecutor, "dev.flutter.echo.basic.binary", BinaryCodec.INSTANCE)
+        val basicBinary = BasicMessageChannel(flutterEngine.dartExecutor, "dev.flutter.echo.basic.binary", BinaryCodec.INSTANCE_DIRECT)
```

Comment:
I tried to find `INSTANCE_DIRECT` in our API docs to see what the difference to `INSTANCE` is, but wasn't able to locate it. Shouldn't it appear here:

https://master-api.flutter.dev/javadoc/io/flutter/plugin/common/BinaryCodec.html

Reply:
It got reverted here https://github.com/flutter/engine/pull/26470.  I'm working on the reland right now.